lib/chromedriver.js

```diff
@@ -143,6 +148,33 @@ class Chromedriver extends events.EventEmitter {
     return cds;
   }
 
+  async getChromeVersion () {
+    let chromeVersion;
+
+    // try out webviews when no bundle id is sent in
+    if (!this.bundleId) {
+      // default to the generic Chrome bundle
+      this.bundleId = CHROME_BUNDLE_ID;
+
+      // we have a webview of some sort, so try to find the bundle version
+      for (const bundleId of WEBVIEW_BUNDLE_IDS) {
+        chromeVersion = await getChromeVersion(this.adb, bundleId);
```

Comment:
recursive call?

Reply:
No. One is a utility function, the other a function on the `Chromedriver` object.